### PR TITLE
chore (v8-vr-tests): Remove usage of Screener

### PR DESCRIPTION
**Pull request by TristanWatanabe**
## Changes:
- removes `Screener` specific configs.
- removes `Screener` related dependencies from package.json.
- Replaces `Screener` tags with `StoryWright` tags,
- Replaces `screener` related imports with `storywright`.
## Related Issue(s)

* Follows #25977
* Part of #25078

### CODE REVIEW

File: apps/vr-tests/package.json [hunk 1]
```diff
@@ -12,7 +12,6 @@
     "just": "just-scripts",
     "lint": "just-scripts lint",
     "screener:build": "cross-env NODE_OPTIONS=--max-old-space-size=3072 just-scripts storybook:build",
```
Comment (TristanWatanabe) [inline apps/vr-tests/package.json:14]: `screener:build` will be renamed in a separate PR along with other miscellaneous changes

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/baf93e56255ce8136a965a7304d7d5bcc0042384'>baf93e56255ce8136a965a7304d7d5bcc0042384</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=25979&commitId=2c8c46ca7bb20a4f2b79be69d96689ab09cb652b&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Review by micahgodbolt — APPROVED** [review]
